packages/monorepo-scripts/src/globals.d.ts

```diff
@@ -1,6 +1,7 @@
 declare module 'async-child-process';
 declare module 'publish-release';
 declare module 'es6-promisify';
+declare module 'semver-diff';
```

Comment:
1. Define types.
2. Move those types to `typescript-typings`

Reply:
I can only do this once there is an older version of `typescript-typings` published... `monorepo-scripts` will otherwise cause a dependency cycle. This is a bit hacky but don't know another way around it.

Reply:
Gotcha. True